firmware/baseband/proc_capture.hpp

```diff
@@ -46,9 +46,13 @@ class CaptureProcessor : public BasebandProcessor {
 
    private:
     // TODO: Repeated value needs to be transmitted from application side.
-    size_t baseband_fs = 0;
+    size_t baseband_fs = 3072000;
```

Comment:
Almost all of the processors default to something sane. I set a sane value for the three that had initialized it to 0.